
libdrm 2.4.104 as of commit https://gitlab.freedesktop.org/mesa/drm/-/commit/cdd14e92e9bd11e7307512b81b9ce26dd5ee2bce 
already adds hdr_metadata_infoframe and hdr_output_metadata structs so disable defining them here


diff -Naur mythtv-master-20210104-gd8376645f6-old/mythtv/libs/libmythtv/drm/mythvideodrmutils.h mythtv-master-20210104-gd8376645f6-new/mythtv/libs/libmythtv/drm/mythvideodrmutils.h
--- mythtv-master-20210104-gd8376645f6-old/mythtv/libs/libmythtv/drm/mythvideodrmutils.h	2021-01-24 18:59:36.000000000 +0100
+++ mythtv-master-20210104-gd8376645f6-new/mythtv/libs/libmythtv/drm/mythvideodrmutils.h	2021-01-24 22:23:48.289998510 +0100
@@ -19,7 +21,7 @@
     HDMI_STATIC_METADATA_TYPE1 = 1,
 };
 
-struct hdr_metadata_infoframe
+/* struct hdr_metadata_infoframe
 {
     uint8_t eotf;
     hdmi_metadata_type metadata_type;
@@ -45,6 +47,7 @@
         struct hdr_metadata_infoframe hdmi_metadata_type1;
     };
 };
+*/
 }
 
 class MythVideoDRMUtils
